src/qmoperators/two_electron/ExchangePotentialD1.cpp

```diff
@@ -122,86 +120,81 @@ void ExchangePotentialD1::setupInternal(double prec) {
  * The exchange potential is (pre)computed among the orbitals that define the operator
  */
 void ExchangePotentialD1::setupInternal_bank(double prec) {
```

Comment:
Note that the main loop in setupInternal_bank will be completely different in next version. (i.e. do not put too much effort to try to improve it)